tests/storage/test_delete.py

```diff
@@ -0,0 +1,65 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import sys
+
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from TestBase import Test
+import argparse
+
+extension = ".txt"
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--file_name", dest="file_name",
+                        required=True,
+                        help='Name of the file')
+    parser.add_argument("--dest", dest="path",
+                        required=True,
+                        help='Specify the path')
+    args = parser.parse_args()
+    return args
+
+
+class Delete(Test):
+    """Checks if a file gets deleted or not"""
+
+    def __init__(self, file_name, dest_path):
+        self.storage_path = dest_path
```

Comment:
Do we really need all these variables? Say, from line #35 to #38
I would assume the user gives me the full path and if the file is not there, the test fails.